CaImAn/caiman_main.py

```diff
@@ -198,10 +198,11 @@ def save_caiman_to_doric(
     shape = np.array(Y).shape
     shape = (shape[1],shape[2], shape[0])
 
-    AC = (A.dot(C)).reshape(shape, order='F').transpose((-1, 0, 1))
-    Y  = Y.reshape(shape, order='F').transpose((-1, 0, 1))
+    # Because of the Fortran ordering it need to be reshaped in a specific shape(..., ..., Time) then transposed to (Time, ..., ...) to be saved
```

Comment:
`# Before saving, the arrays have to be reshaped with Fortran order and shape (..., ..., Time), then transposed to (Time, ..., ...)`

Reply:
what are (..., ..., Time) and (Time, ..., ...)? (Width, Height, Time) or (Height, Width, Time), ...?